docs/FAQs.md

```diff
@@ -21,11 +21,16 @@ Provisioners are heterogeneous, which means that the nodes they manage are sprea
 ### What if my pod is schedulable for multiple Provisioners?
 It's possible that an unconstrained pods could flexibly schedule in multiple groups. In this case, Provisioners will race to create a scheduling lease for the pod before launching new nodes, which avoids unnecessary scale out.
 ## Reallocation
-TODO
+### How does Karpenter terminate nodes?
+Karpenter annotates nodes that are underutilized with a TTL. If the node remains underutilized after the TTL expires, Karpenter then cordons the node and uses the Kubernetes Eviction API to evict all non-daemonset pods. Once the node is empty, the node is terminated.
+### How does Karpenter decide which nodes it can terminate? 
+A provisioner will only take action on nodes that it manages. This means that a node will only be considered for termination if it labeled underutilized by the provisioner that manages it.
+### How do I know if a node is underutilized?
+Nodes are considered underutilized if they have 0 non-daemon set pods scheduled. We plan to include more use cases in the future.
```

Comment:
nit: `non-daemon set` -> `non-daemonset`   it seems to be used as one word elsewhere in this doc and in the community (`kubectl drain --ignore-daemonsets`) even though it is camel-cased when it is written in uppercase (i..e DaemonSet).